.github/workflows/okta-react-native.yml

```diff
@@ -24,17 +24,17 @@ jobs:
       - name: Execute snyk.sh
         run: bash ./scripts/snyk.sh
     iOSUnitTests:
-      runs-on: macos-11
+      runs-on: macos-12
```

Comment:
Had to update this to macos-12 because iosUnitTests randomly started failing without making any changes to iOS files. As a result, I also had to change the device for iosUnitTests to iPhone 14 since iPhone 12 isn't available after update to macos-12